include/internal/libspdm_requester_lib.h

```diff
@@ -129,10 +129,37 @@ libspdm_return_t libspdm_get_version(libspdm_context_t *spdm_context,
                                      uint8_t *version_number_entry_count,
                                      spdm_version_number_t *version_number_entry);
 
+#define LIBSPDM_MAX_EXT_ALG_COUNT 5
```

Comment:
libspdm does not support ext_alg. I think we can ignore them totally.

Reply:
No, I don't suggest to keep the complexity.

As we already said, libspdm does not support ext_alg.
What is the value to return ext_alg to the integrator?
The integrator cannot use ext_algo with libspdm anyway.